libraries/botbuilder-dialogs-declarative/src/resources/resourceProvider.ts

```diff
@@ -42,6 +42,10 @@ export abstract class ResourceProvider {
      * Event which is fired if any resource managed by the resource provider detects changes to the underlining resource.
      * @param callback Callback function to be called when an event fired.
      */
+    public get changed(): (event: ResourceChangeEvent, resources: Resource[]) => void {
+        return;
+    }
+
```

Comment:
Note: added to fix

```cmd
Error: src/resources/resourceExplorer.ts:72:5 - (ae-missing-getter) The property "changed" has a setter but no getter.
 | Error: src/resources/resourceProvider.ts:45:5 - (ae-missing-getter) The property "changed" has a setter but no getter.
```

Reply:
We should probably configure this error to be a warning only, there's no reason we have to expose properties with getters.

Reply:
If a warning is shown in the console, [it still throws a non-zero exit code](https://api-extractor.com/pages/setup/configure_api_report/), so I made them "warnings" and redirected the output to the `.md` file. So unfortunately, the warnings can really only be seen when diff-ing the .md files.